lib/jazzy/sourcekitten.rb

```diff
@@ -409,7 +409,7 @@ def self.filter_excluded_files(json)
       excluded_files = Config.instance.excluded_files
       json.map do |doc|
         key = doc.keys.first
-        doc[key] unless excluded_files.include?(Pathname(key))
+        doc[key] unless excluded_files.detect{|file| key.include? file.to_s}
```

Comment:
no particular reason. Just didn't think of it :) I can make the change if you'd like.